### PR TITLE
Refactor instances of master to control plane 

### PR DESCRIPTION
**Pull request by mgutierrez98**
#### What type of PR is this?
/kind cleanup
/wg naming
/sig testing

#### What this PR does / why we need it:
Refactor names within stats.go from master to control plane

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs

```

### CODE REVIEW

**Comment by mgutierrez98** [comment]
Only meant to push one file change and will fix this PR to refect the stats.go change only.

File: test/e2e/framework/kubelet/stats.go [hunk 1]
```diff
@@ -548,14 +548,14 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
 	return result, utilerrors.NewAggregate(errs)
 }
 
-// GetMasterNodeLatest returns the latest resource usage of master and node.
-func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
+// GetControlPlaneNodeLatest returns the latest resource usage of control plane and node.
+func (r *ResourceMonitor) GetControlPlaneNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
 	result := make(ResourceUsagePerNode)
```
Comment (wangyysde) [inline test/e2e/framework/kubelet/stats.go:553]: here you rename GetMasterNodeLatest to GetControlPlaneNodeLatest. But I can not find where is calling GetControlPlaneNodeLatest

**Comment by wangyysde** [comment]
As if you should squashing your commits.

File: test/e2e/framework/kubelet/stats.go (hunk 1 above)
Comment (mgutierrez98) [reply test/e2e/framework/kubelet/stats.go:553]: Thank you for pointing this out. I will find the instance of GetMasterNodrLatest and refactoring it for control plane usage when I get back to my computer.

**Pull request closed without merging**